pinax/stripe/models.py

```diff
@@ -24,8 +24,16 @@ class Meta:
         abstract = True
 
 
+class AccountRelatedStripeObject(StripeObject):
```

Comment:
@iMerica - "specific models that need the field can use it" - this is the intent. Models that can be account specific (which isn't all of them - it can be redundant to make them all partitioned by account) inherit from the AccountRelatedStripeObject abstract model.

Let me know if I've missed your point.